lib/ContractInteract/BrandedToken.js

```diff
@@ -0,0 +1,393 @@
+   *                                  instance that has been deployed.
+   */
+  static async deploy(
+    web3,
+    valueToken,
+    symbol,
+    name,
+    decimals,
+    conversionRate,
+    conversionRateDecimals,
+    organization,
+    txOptions,
+  ) {
+    if (!txOptions) {
+      const err = new TypeError('Invalid transaction options.');
+      return Promise.reject(err);
+    }
+    if (!Web3.utils.isAddress(txOptions.from)) {
+      const err = new TypeError(`Invalid from address: ${txOptions.from}.`);
+      return Promise.reject(err);
+    }
+
+    const tx = BrandedToken.deployRawTx(
+      web3,
+      valueToken,
+      symbol,
+      name,
+      decimals,
+      conversionRate,
+      conversionRateDecimals,
+      organization,
+    );
+
+    return Utils.sendTransaction(tx, txOptions).then((txReceipt) => {
+      const address = txReceipt.contractAddress;
+      return new BrandedToken(web3, address);
+    });
+  }
+
+  /**
+   * Raw transaction for {@link BrandedToken#deploy}.
+   *
+   * @param web3 Origin chain web3 object.
+   * @param valueToken The value to which valueToken is set.
+   * @param symbol The value to which tokenSymbol, defined in EIP20Token,
+   *                is set.
+   * @param name The value to which tokenName, defined in EIP20Token,
+   *              is set.
+   * @param decimals The value to which tokenDecimals, defined in EIP20Token,
+   *                  is set.
+   * @param conversionRate The value to which conversionRate is set.
+   * @param conversionRateDecimals The value to which
+   *                                conversionRateDecimals is set.
+   * @param organization Organization contract address.
+   *
+   * @returns {Object} Raw transaction.
+   */
+  static deployRawTx(
+    web3,
+    valueToken,
+    symbol,
+    name,
+    decimals,
+    conversionRate,
+    conversionRateDecimals,
+    organization,
+  ) {
+    if (!(web3 instanceof Web3)) {
+      throw new TypeError(
+        `Mandatory Parameter 'web3' is missing or invalid: ${web3}`,
+      );
+    }
+    if (!Web3.utils.isAddress(valueToken)) {
+      throw new TypeError(`Invalid valueToken address: ${valueToken}.`);
+    }
+    if (!Web3.utils.isAddress(organization)) {
+      throw new TypeError(`Invalid organization address: ${organization}.`);
+    }
+    if (!(conversionRate > 0)) {
+      throw new TypeError(`Invalid conversion rate: ${conversionRate}. It should be more than zero`);
+    }
+    if (!(conversionRateDecimals < 5)) {
+      throw new TypeError(`Invalid conversion rate decimal: ${conversionRateDecimals}. It should be less than 5`);
+    }
+
+    const abiBinProvider = new AbiBinProvider();
+    const bin = abiBinProvider.getBIN(ContractName);
+
+    const args = [
+      valueToken,
+      symbol,
+      name,
+      decimals,
+      conversionRate,
+      conversionRateDecimals,
+      organization,
+    ];
+
+    const contract = Contracts.getBrandedToken(web3, null, null);
```

Comment:
Are you sure you want to pass in `null` and not `undefined` (or simply remove the arguments)? Why `null`?